### DISCUSSION TITLE
Les fonctions parse_tennis() et parse_football() ne sont pas disponibles

### DISCUSSION THREAD

**Discussion by AlexLA92** [pretrehr/Sports-betting#45] [General]
Bonjour, 

J'ai bien installé les requirements et suivi les étapes d'initialisation du ReadMe mais je n'arrive pas à appeler les fonctions parse_tennis() et parse_football() ( erreur : NameError: name 'parse_tennis' is not defined
). Pourtant ces fonctionnalités sont disponibles via l'interface graphique. 
Puis-je accéder au résultat du parser de cotes tennis en utilisant une autre fonction, comme parse_competition par exemple ?

**Comment by augbel72** [comment]
Bonjour, (j'utilise le traducteur Google donc je m'excuse pour les erreurs) J'ai moi aussi le même problème signalé ci-dessus : la fonction "parse_football" n'existe pas ! Est-ce que quelqu'un sait comment obtenir les cotes uniquement pour certains marchés (par exemple 1X2, Moins/Plus de 1,5/2,5/3,5, Double Chance) et PAS seulement 1x2 pour tous les matchs (avant-match) de tous les événements de football ?
